src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -86,5 +86,4 @@ public Key(T defaultValue) {
        Object value = store.get(key);
        return value != null ? (T)value : key.defaultValue;
```

Comment:
That feels weird but i will include it. If there are any other styling issues you see let me know. :)